index.html

```diff
@@ -4,9 +4,83 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Who Travels?</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat+Alternates&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/assets/css/style.css">
 </head>
 <body>
-    
+    <header>
+        <section class="header">
+            <img src="#" alt="Who Travels Logo" class="logo">
+            <nav>
+                <div class="mobile-menu">
+                    <input type="checkbox" />
+                    <span class="hamburger"></span>
+                    <span class="hamburger"></span>
+                    <span class="hamburger"></span>
+
+                    <ul class="menu">
+                        <li><a href="book.html" class="top-nav" id="book">Book Trip</a></li>
+                        <li><a href="destinations.html" class="top-nav" id="destinations">Destinations</a></li>
+                        <li><a href="about.html" class="top-nav" id="about">About Us</a></li>
+                        <li><a href="contact.html" class="top-nav" id="contact">Contact</a></li>
+                        <li><a href="explore.html" class="top-nav" id="explore">Explore</a></li>
+                    </ul>
+
+                </div>
+            </nav>
+        </section>
+    </header>
+
+    <h1>Who Travels?</h1>
+    <p>It could be you! The one stop shop for all time and space travel needs.</p>
+
+    <p>From Pompeii in 79 A.D. to the Sapphire Waterfall of planet Midnight, Who Travels? offers a stunning array of adventures across time and space. See what our guests have to say about their favorite locals. </p>
+    <p>Explore a place in time or space</p>
+    <form>
+        <label for="form-destination">Destination:</label><br>
+        <select id="cars" name="cars">
```

Comment:
I think this was misnamed `cars` instead of `cards` maybe give a more descriptive name such as `home-destination-slides` or `home-destination-cards`. I would also change the id to a class

Reply:
No, it's because I was dumb and tired and didn't notice I hadn't edited it correctly